python/desc/twinkles/twinkles_sky.py

```diff
@@ -19,10 +19,10 @@
 from lsst.sims.catUtils.exampleCatalogDefinitions import\
     (PhoSimCatalogPoint,
      PhoSimCatalogSersic2D,
-     PhoSimCatalogSN,
      DefaultPhoSimHeaderMap,
      DefaultPhoSimInstanceCatalogCols)
 from .twinklesCatalogDefs import TwinklesCatalogZPoint
+from .twinklesCatalogDefs import TwinklesPhoSimCatalogSN as PhoSimCatalogSN
```

Comment:
I would rather import this as TwinklesPhoSimCatalogSN and then change the use of PhoSimCatalogSN to TwinklesPhoSimCatalogSN, so that, when looking at the body of the code, it is obvious that we are not using the sims_catUtils version of PhoSimCatalogSN